example-app/src/main/java/com/example/app/OTAUpdateActivity.java

```diff
@@ -0,0 +1,141 @@
+package com.example.app;
+
+import static com.example.app.Dialogs.hideProgress;
+import static com.example.app.Dialogs.showProgress;
+import static com.example.app.Dialogs.updateProgress;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.app.AlertDialog;
+import android.bluetooth.BluetoothDevice;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+
+import com.tsys.payments.library.enums.TerminalUpdateType;
+import com.tsys.payments.library.exceptions.Error;
+import com.tsys.payments.library.terminal.AvailableTerminalVersionsListener;
+import com.tsys.payments.library.terminal.UpdateTerminalListener;
```

Comment:
@philip-white-heartland We'll want to make sure that any types used by the integrator are exported by a Heartland branded package for consistency.